docs(ldap-adv) add new group mapping properties (#1756)
Kong/docs.konghq.com: app/enterprise/1.3-x/kong-manager/service-directory-mapping.md

    * For example, `ec2-XX-XXX-XX-XXX.compute-1.amazonaws.com`
* `ldap_password`:`<ENTER_YOUR_LDAP_PASSWORD_HERE>`: LDAP password
    * *Important*: As with any configuration property, sensitive information may be set as an environment variable instead of being written directly in the configuration file.
* `group_base_dn`:`<ENTER_YOUR_BASE_DN_HERE>`: Sets a distinguished name for the entry where LDAP searches for groups begin. The default is the value from `conf.base_dn`.

> * `group_name_attribute`: `<ENTER_YOUR_GROUP_NAME_ATTRIBUTE_HERE>`: Sets the attribute holding the name of a group, typically called name (for example, for Active Directory) or cn (for example, for OpenLDAP). The default is the value from `conf.attribute`.

Reviewer: This is a bit confusing....can the "for" be removed after "for example"?   ...."typically called name (for example, Active Directory) or cn (for example, OpenLDAP). The default is the value from `conf.attribute`.

Author: ```suggestion
* `group_name_attribute`: `<ENTER_YOUR_GROUP_NAME_ATTRIBUTE_HERE>`: Sets the attribute holding the name of a group, typically called `name` (in Active Directory) or `cn` (in OpenLDAP). The default is the value from `conf.attribute`.
```
